### DISCUSSION TITLE
How do I temporarily show hidden files?

### DISCUSSION THREAD

**Discussion by chrisolsen** [nvim-neo-tree/neo-tree.nvim#596] [Q&A] [answered]
I can't find any keybindings in the help menu that indicates that this can be done, is this doable?

Thanks

**Comment by cseickel** [comment, marked as answer]
`H` = `toggle_hidden`

**Reply by chrisolsen** [reply]
For me `H` cycles through the File, Bufs and Git tabs
![image](https://user-images.githubusercontent.com/41582/200456869-e2280a8f-ac23-443c-81a5-db28e4553a87.png)

**Reply by cseickel** [reply]
Did you configure custom mappings? Pressing `?` in the neo-tree window should bring up the list of defaults. If you don't see the issue in your config, post or link to it here.

**Reply by chrisolsen** [reply]
I am using AstroVim and recently they switched from nvim-tree to neo-tree and the keymapping changed, but I didn't realize they added custom keymapping on top of that.

Thanks for the help
